### PR TITLE
Week five winner announcement

### PR DESCRIPTION
**Pull request by johnnewton-hub**
Hi Mark,
Here's a quick announcement for Open Marketplace's win.

### CODE REVIEW

File: _posts/2020-04-21-week5-winner.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+author: John
+title: Congratulations Week Five Winner: Open Marketplace!
+---
+
+Our week five winner [Open Marketplace](https://open-marketplace-applications.github.io/marketplace/), is leveraging the open source and nonprofit model used by Wikipedia to improve Covid-19 response.
+
+<img src="/images/blog/open marketplace.jpg" alt="Get Cleard" style="max-width: 100%;">
```
Comment (MarkBennett) [inline _posts/2020-04-21-week5-winner.md]: Can you change the alt text here to Open Marketplace?

**Review by MarkBennett — CHANGES_REQUESTED** [review]
Looks good. One small tweak.

File: _posts/2020-04-21-week5-winner.md (hunk 1 above)
Comment (johnnewton-hub) [reply _posts/2020-04-21-week5-winner.md]: dough!  cut and paste can be so dangerous

**Review by johnnewton-hub — COMMENTED** [review]
that should be it...

**Review by MarkBennett — APPROVED** [review]
Nice work @johnny-newton!

**Comment by johnnewton-hub** [comment]
Will squash and merge then!  This is a personal best..for now : )

**Pull request merged**